crates/nargo_cli/tests/execution_success/slice_dynamic_index/src/main.nr

```diff
@@ -0,0 +1,191 @@
+fn main(x : Field) {
+    assert(x == 5);
+    // The parameters to this function must come from witness values (inputs to main).
+    // The assert on `x` before this call is to guarantee we test the expected
+    // conditional branches.
```

Comment:
Heads up that I've got a local branch which adds an optimization which will result in all instances of `x` being replaced with the constant `5` after we assert it like so.

Reply:
I removed this assert